### PR TITLE
Adjust the molecule cf https://github.com/ome/ansible-role-ssl-certif…

### PR DESCRIPTION
**Pull request by pwalczysko**
Attempting to fix the build leaning on https://github.com/ome/ansible-role-ssl-certificate/pull/17 in first iteration

cc @sbesson @jburel

### CODE REVIEW

**Comment by pwalczysko** [comment]
Not sure why the GA is not showing here on this PR, but I have this PR green on https://github.com/pwalczysko/ansible-role-basedeps/actions/runs/5277814391

cc @sbesson @jburel

**Comment by pwalczysko** [comment]
I know now why the actions do not show here. This is because they were disabled due to inactivity automatically.

My experience when clicking on GA tab and then clicking on "Molecule" in the LHP:
Screenshot 2023-06-15 at 11 58 06

--------------------------------

The github action docs say there should be an "Enable" button, which is missing for me, I guess because of perms issues ?
Screenshot 2023-06-15 at 11 55 26

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -3,13 +3,13 @@ dependency:
...
+lint: |
+    yamllint .
+    ansible-lint
+    flake8
 platforms:
   - name: centos-7
-    image: centos:7
-  - name: ubuntu-1804
-    image: ubuntu:18.04
+    image: centos/systemd
```
Comment (sbesson) [inline molecule/default/molecule.yml:12]: Any reason to use the systemd image here? There should be no requirement for services to be started

**Review by sbesson — COMMENTED** [review]
One question about the base image used for testing. No objection otherwise

File: molecule/default/molecule.yml (hunk 1 above)
Comment (pwalczysko) [reply molecule/default/molecule.yml:12]: Yes, the reason is
``fatal: [centos-7]: FAILED! => {"changed": false, "msg": "Failure talking to yum: Cannot retrieve metalink for repository: epel/x86_64. Please verify its path and try again"}`` error in converge block of https://github.com/pwalczysko/ansible-role-basedeps/actions/runs/5271513129/jobs/9532500450 -  the switch to centos/systemd fixed this error

**Review by sbesson — COMMENTED** [review]
Looks good to me. The only other annotation raised by the actions is

```
the role 'ansible-role-basedeps' was not found in /home/runner/work/ansible-role-basedeps/ansible-role-basedeps/molecule/default/roles:/home/runner/.cache/ansible-compat/6fbc65/roles:/home/runner/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/home/runner/work/ansible-role-basedeps/ansible-role-basedeps/molecule/default
```

in `molecule/default/playbook.yml` which I assume could be updated to `ome.basedeps`. Would you prefer to handle this here or separately?

**Comment by pwalczysko** [comment]
> in `molecule/default/playbook.yml` which I assume could be updated to `ome.basedeps`. Would you prefer to handle this here or separately?

This error and four other errors (all of FQCN nature) are fixed now by https://github.com/ome/ansible-role-basedeps/pull/11/commits/7951c6a1bb0b950d2ee5d7bc803d36a35e83c252 and https://github.com/ome/ansible-role-basedeps/pull/11/commits/d2e6df2ea49084a2a4263a3a2dfa85fc4cf9457b (for some reason the 4 errrors fixed in the last commit appeared only after the fix of the error/annotation remarked on by you

**Review by sbesson — APPROVED** [review]

**Comment by pwalczysko** [comment]
@jburel Do you want to merge ?

**Pull request merged**